web/src/features/hfiCalculator/components/DayHeaders.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { Table, TableBody, TableContainer, TableRow } from '@material-ui/core'
+import { render } from '@testing-library/react'
+import React from 'react'
+import DayHeaders from 'features/hfiCalculator/components/DayHeaders'
+describe('DayHeaders', () => {
+  it('should return table row with the headers for Monday - Friday given the ISO Date', () => {
+    const prepCycle = [
+      'Mon., Oct. 04',
+      'Tue., Oct. 05',
+      'Wed., Oct.06',
```

Comment:
Is this working? There's no space between "Oct." and "06" - why is this passing?

Reply:
Fixed!